libs/datasets/dataset_utils.py

```diff
@@ -18,7 +18,7 @@
     CommonFields.COUNTY,
 ]
 
-NON_NUMERIC_COLUMNS = GEO_DATA_COLUMNS + [CommonFields.CAN_LOCATION_PAGE_URL]
+NON_NUMERIC_COLUMNS = GEO_DATA_COLUMNS + [CommonFields.CAN_LOCATION_PAGE_URL, CommonFields.HSA]
```

Comment:
So I think the HSA column was being converted to numerics in the [`_merge_attributes()` method](https://github.com/covid-projections/covid-data-model/blob/main/libs/datasets/timeseries.py#L391-L392) when being loaded into a `MultiRegionDataset`, so specifying this column as non-numeric should hopefully solve the issue

Reply:
Nvm, that didn't work :/ ([see this snapshot](https://data.covidactnow.org/snapshot/3107/v2/county/36061.timeseries.json))

Reply:
977caf12cc058f3f994cc007bdca73a3102c7c8d fixes this in a kind of hacky way